apps/website/src/pages/docs/dialog.mdx

```diff
@@ -13,58 +17,53 @@ import LiveExample from '~/components/LiveExample.astro';
-- Close button: only allowed in dismissible dialogs.
-- Content
-- Actions: see Button standard for best practices.
-- Box & shadow
-- Overlay: Dialog appears above all other UI elements and casts a black overlay as well as a drop shadow on all other content, creating a strong visual focus on the dialog.
-
-The dialog should be located in the upper half of the screen, because on smaller screens it may get lost. A dialog’s maximum width is 50% of the screen width & the minimum width is 400px. If your content can not fit inside the dialog without scrollbars, you should consider alternative ways to display the information.
+## Types
 
-When the 400px minimum width conflicts with the 50% maximum of the screen width, then the dialog spans the entire width of the device. This mobile dialog can span the entire height of the screen but should not use scroll bars.
+### Dismissible
 
-## Types
+A dismissible dialog contains information that is not required to complete before being able to proceed.
 
-### Dismissible dialogs
+<LiveExample src={frontmatter.exampleCodeDismissible} />
 
-A dismissible dialog contains information that is not required. The dialog can be dismissed multiple different ways:
+The dialog can be dismissed multiple different ways:
 
 - Clicking one of the action buttons.
 - Clicking on “X” button.
-- Pressing the ESC key on your keyboard.
-- Clicking elsewhere outside the dialog window.
+- Pressing the <kbd>ESC</kbd> key on your keyboard.
+- Clicking outside the dialog window.
 
 The “X” icon bears the borderless button style which appears on cursor hover.
 
-### Non dismissible dialogs
+### Non-dismissible
 
 A non dismissible dialog contains important information that must be completed. The only way to dismiss a non dismissible dialog is by completing the action.
 
-### Drag-enabled dialogs
+<LiveExample src={frontmatter.exampleCodeNonDismissible} />
 
-These dialogs have a definite header that can be clicked and moved around the screen at the user’s leisure, just like a browser window.
+### Draggable
 
-### Full page dialogs
+These dialogs have a defined header that can be clicked and moved around the screen at the user’s leisure, just like a browser window. This dialog can also be resized.
+
+<LiveExample src={frontmatter.exampleCodeDraggable} />
+
+### Full page
 
 A full page dialog appears on top of the content and occupy the entire page’s space. It is used primarily for pages where users will customize settings.
 
+<LiveExample src={frontmatter.exampleCodeFullpage} />
+
 The two buttons at the bottom right are always pinned in that same area, regardless of content. If there is more content than there is available space in the content area, a scrollbar can be included to make the content area scrollable.
 
 As specified earlier, full-page dialogs will house various settings options as shown in the example below.
 
 ## Usage guidelines
 
-### Dos & Don'ts
-
 Do:
```

Comment:
I left Do & Don't as is so they aren't added to the table of contents.  That makes sense to me, but if others feel otherwise I have no issue with changing.